examples/xds-sotw-config-server/logger.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020 Envoyproxy Authors
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
```

Comment:
We don't put a license on top of every file so you can remove this here and elsewhere.

Reply:
Removed with the commit https://github.com/envoyproxy/ratelimit/pull/373/commits/1873901a3a5c42874c8c2f704b8fed43c5227ba7